release/ray_release/config.py

```diff
@@ -20,6 +20,7 @@ class Test(dict):
 DEFAULT_BUILD_TIMEOUT = 1800
 DEFAULT_CLUSTER_TIMEOUT = 1800
 DEFAULT_AUTOSUSPEND_MINS = 120
+DEFAULT_MAXIMUM_UPTIME_MINS = 3200
```

Comment:
this is is still ~53.3 hrs > 2 days however, do we really have tests intended to run for that long ?